XCUITests/DataManagementTest.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  DataManagementTest.swift
+//  XCUITests
+//
+//  Created by Meera Rachamallu on 9/4/18.
+//  Copyright © 2018 Mozilla. All rights reserved.
+//
+
+import XCTest
+
+class DataManagementTest: BaseTestCase {
+    func testCheckDataManagementSettingsByDefault() {
+        navigator.goto(WebsiteDataSettings)
+        print(app.debugDescription)
+        waitforExistence(app.navigationBars["Website Data"])
+
+        navigator.performAction(Action.AcceptClearAllWebsiteData)
```

Comment:
Since this test is called: checkDataManagementSettingsByDefault, I would not expect to clear the data here, just to check the settings and then clear the data in another test where you previously generate some data to remove. Does it make sense to you?

Reply:
yes it does!